src/neo/SmartContract/ContractState.cs

```diff
@@ -0,0 +1,57 @@
+using Neo.IO;
+using Neo.IO.Json;
+using Neo.SmartContract.Manifest;
+using Neo.VM;
+using Neo.VM.Types;
+using System;
+using System.Linq;
+using Array = Neo.VM.Types.Array;
+
+namespace Neo.SmartContract
+{
+    public class ContractState : IInteroperable
+    {
+        public int Id;
+        public ushort UpdateCounter;
+        public UInt160 Hash;
+        public byte[] Script;
+        public ContractManifest Manifest;
+
+        void IInteroperable.FromStackItem(StackItem stackItem)
+        {
+            Array array = (Array)stackItem;
+            Id = (int)array[0].GetInteger();
+            UpdateCounter = (ushort)array[1].GetInteger();
```

Comment:
Now maybe we don't need to cast to ushort, because it will be serialized as `BigInteger`

Reply:
There is no difference. BigInteger is optimized for the value smaller then int.